tools/seed_data/seed_vuln_data.go

```diff
@@ -0,0 +1,351 @@
+	Hostname         string     `db:"hostname"`
+	Uuid             string     `db:"uuid"`
+	Platform         string     `db:"platform"`
+	OsqueryVersion   string     `db:"osquery_version"`
+	OsVersion        string     `db:"os_version"`
+	Build            string     `db:"build"`
+	PlatformLike     string     `db:"platform_like"`
+	CodeName         string     `db:"code_name"`
+	Uptime           int64      `db:"uptime"`
+	Memory           int64      `db:"memory"`
+	CpuType          string     `db:"cpu_type"`
+	CpuSubtype       string     `db:"cpu_subtype"`
+	CpuBrand         string     `db:"cpu_brand"`
+	CpuPhysicalCores int        `db:"cpu_physical_cores"`
+	CpuLogicalCores  int        `db:"cpu_logical_cores"`
+	HardwareVendor   string     `db:"hardware_vendor"`
+	HardwareModel    string     `db:"hardware_model"`
+	HardwareVersion  string     `db:"hardware_version"`
+	HardwareSerial   string     `db:"hardware_serial"`
+	ComputerName     string     `db:"computer_name"`
+	PrimaryIP        string     `db:"primary_ip"`
+	PrimaryMac       string     `db:"primary_mac"`
+	LabelUpdatedAt   time.Time  `db:"label_updated_at"`
+	LastEnrolledAt   time.Time  `db:"last_enrolled_at"`
+	RefetchRequested int        `db:"refetch_requested"`
+	PublicIP         string     `db:"public_ip"`
+}
+
+type HostDisplayName struct {
+	HostID int64  `db:"host_id"`
+	Name   string `db:"display_name"`
+}
+
+type Software struct {
+	ID               int64  `db:"id"`
+	Name             string `db:"name"`
+	Version          string `db:"version"`
+	Source           string `db:"source"`
+	BundleIdentifier string `db:"bundle_identifier"`
+	Release          string `db:"release"`
+	VendorOld        string `db:"vendor_old"`
+	Arch             string `db:"arch"`
+	Vendor           string `db:"vendor"`
+}
+
+type HostSoftware struct {
+	HostID     int64 `db:"host_id"`
+	SoftwareID int64 `db:"software_id"`
+}
+
+func readCSVFile(filePath string) ([]Software, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	reader := csv.NewReader(file)
+	reader.Comma = ',' // Set the delimiter to comma
+	lines, err := reader.ReadAll()
+	if err != nil {
+		return nil, err
+	}
+
+	var softwares []Software
+	for _, line := range lines[1:] { // Skip header line
+		softwares = append(softwares, Software{
+			Name:             line[0],
+			Version:          line[1],
+			Source:           line[2],
+			BundleIdentifier: line[3],
+			Release:          line[4],
+			VendorOld:        line[5],
+			Arch:             line[6],
+			Vendor:           line[7],
+		})
+	}
+	return softwares, nil
+}
+
+func insertOrUpdateSoftware(db *sqlx.DB, hostID int, software Software) error {
+	// Check if the software already exists
+	var existingID int64
+	query := `SELECT id FROM software WHERE name = ? AND version = ? AND source = ?`
+	err := db.Get(&existingID, query, software.Name, software.Version, software.Source)
+	if err != nil && err != sql.ErrNoRows {
```

Comment:
```suggestion
	if err != nil && errors.Is(err, sql.ErrNoRows) {
```
I think `errors.Is` should generally be used, since it can also handle wrapped errors.